settings/api/api.go

```diff
@@ -45,7 +46,7 @@ func main() {
 	ctrl := controller.New(model)
 	routes.MakeRoutes(router, ctrl)
 
-	log.Printf("Socrates API serving on PORT %s", viper.GetString("API_PORT"))
+	fmt.Printf("Socrates API serving on PORT %s", viper.GetString("API_PORT"))
```

Comment:
Don't use fmt. Use log.Info

Reply:
This is only here to help me debug, all the useful info is being logged. Hence the `fmt.Printf`